cmd/gnodev/util.go

```diff
@@ -104,3 +105,13 @@ func guessRootDir() string {
 	rootDir := strings.TrimSpace(string(out))
 	return rootDir
 }
+
+func makeTestGoMod() (string, error) {
+	var out bytes.Buffer
+
+	_, err := out.WriteString("module github.com/gnolang/gno\n\ngo 1.18\n")
```

Comment:
These string values (module name and go version) should be extracted out into constants, it makes it easier to change down the road when we bump go versions or move around the packages

Reply:
Resolving this convo as you've rewritten the implementation